### PR TITLE
Add new extras check (7130) to check encryption of a SNS topic

### PR DESCRIPTION
**Pull request by mpratsch**
Hi, 
I was missing the check if a SNS topic is encrypted or not.
That was an outcome of a pen test.
I'd love to see that integrated in prowler as well.

Example of a query only for eu-central-1:
image

Thanks Martina

### CODE REVIEW

**Comment by toniblyx** [comment]
Thanks @mpratsch, please review my question there and also add that check to `groups/group7_extras`. Good job!

**Comment by mpratsch** [comment]
> 
> 
> Thanks @mpratsch, please review my question there and also add that check to `groups/group7_extras`. Good job!

Thanks @toniblyx! Just a stupid question but where do I find your question? 🤔

**Review by toniblyx — CHANGES_REQUESTED** [review]

**Comment by toniblyx** [comment]
I added a couple of comments to the code but I forgot to request changes, now you should see it, sorry about that :)

File: checks/check_extra7130 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        SHORT_TOPIC=$(echo $topic | awk -F ":" '{print $NF}')
+        SNS_ENCRYPTION=$($AWSCLI sns get-topic-attributes --region $regx --topic-arn $topic --query 'Attributes.KmsMasterKeyId' --output text)
+        if [[ "None" == $SNS_ENCRYPTION ]]; then
+          textFail "$regx: $SHORT_TOPIC is not encrypted!" "$regx"
+        else
+          textPass "$regx: $SHORT_TOPIC is encrypted" "$regx"
+        fi
+      done
+    else
+      textInfo "$regx: No SNS topic found" "$SHORT_TOPIC" "$regx"
```
[the remark this answers is not in the document]
Comment (mpratsch) [reply checks/check_extra7130]: Ha, I had no account without a SNS topic and didn't see that one. 
It's not needed and wouldn't even work.
Thanks for that!

**Comment by toniblyx** [comment]
Awesome, thanks @mpratsch!

**Pull request merged**